### PR TITLE
S5 pokemon status styling

### PR DESCRIPTION
**Pull request by f-gate**
added classes to pokemon listings dependant on status from DB
added checked functionality to radio buttons based on DB

file to review : 
Pokemon.php
pokemonEntryTemplate.phtml

### CODE REVIEW

File: dataScrape/src/apiPull.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<?php
+
+class apiPull
```
Comment (hoptond) [inline dataScrape/src/apiPull.php]: Should be capitalised.

File: dataScrape/src/pokemonTableUpdate.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<?php
+
+class pokemonTableUpdate
```
Comment (hoptond) [inline dataScrape/src/pokemonTableUpdate.php]: See above.

File: src/PokeList.php [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: academy
+ * Date: 14/11/2018
+ * Time: 11:48
+ */
```
Comment (hoptond) [inline src/PokeList.php]: Remove this autogenerated comment, it's been removed in the other one

**Review by hoptond — CHANGES_REQUESTED** [review]

File: src/PokeList.php (hunk 3 above)
Comment (f-gate) [reply src/PokeList.php]: done in 79576f6

File: dataScrape/src/pokemonTableUpdate.php (hunk 2 above)
Comment (f-gate) [reply dataScrape/src/pokemonTableUpdate.php]: done in 79576f6

File: dataScrape/src/apiPull.php (hunk 1 above)
Comment (f-gate) [reply dataScrape/src/apiPull.php]: done in 
8f0e5de

**Review by hoptond — APPROVED** [review]

**Pull request merged**